ios/Encryption/EncryptionUtil.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+import CommonCrypto
+
+class EncryptionUtil {
+    enum Error: Swift.Error {
+      case generateRandomFailed
+      case getKeyFailed
+      case decryptionFailed
+    }
+
+    static let KEY_SIZE = kCCKeySizeAES256
+    static let HMAC_SIZE = Int(CC_SHA256_DIGEST_LENGTH)
+    
+    static let AES_KEY_TAG = "covidsafe.keys.aes_key"
+    static let HMAC_KEY_TAG = "covidsafe.keys.hmac_key"
+
+    static func encrypt(plainText: String) -> String {
+        do {
+            let aesKey = try KeyChain.get(tag: AES_KEY_TAG) ?? createKey(tag: AES_KEY_TAG)
+            print("AES key: \(aesKey.base64EncodedString())")
```

Comment:
While knowing this during debugging is useful, should we really print it out during production?